### DISCUSSION TITLE
I have a problem making pyvistaqt work under PyQt6

### DISCUSSION THREAD

**Discussion by mkondratyev85** [pyvista/pyvista#3491] [Q&A] [answered]
A few days ago I've opened an issue in pyvistaqt repository asking for the help on making pyvistaqt work with PyQt6:

https://github.com/pyvista/pyvistaqt/issues/239

Unfortunately I haven't received a reply yet so I'm asking here maybe you know what may cause the problem when I switch from PyQt5 to PyQt6 when using pyvistaqt?

**Comment by banesullivan** [comment, marked as answer]
Hi @mkondratyev85, I'm sorry you haven't heard back. The Qt "flavor/variant" of PyVista is maintained by a small subset of the active people here -- unfortunately, I'm not the most versed on the state of PyVsita + Qt

I'll cc @pyvista/pyvistaqt-admin to see if any of them have a solution for you, but otherwise we may just have to wait and see if someone else working with PyVistaQt has tried out PyQt6

**Reply by mkondratyev85** [reply]
Thank you for the help @banesullivan.

**Reply by adam-grant-hendry** [reply]
@mkondratyev85 I've had some time to work with `pyvistaqt` and am happy to help you. I wrote some suggestions down of how we might start debugging this in your issue. If you can reply there, I'll do my best to help you.
